e2e-test/cypress/support/helper.js

```diff
@@ -2,31 +2,31 @@ import _isString from "lodash/isString";
 
 import accounting from "accounting";
 
-const numberFormat = {
+const numberFormatEnglish = {
   decimal: ".",
   thousand: ",",
   precision: 2
 };
 
-const currencies = {
-  EUR: { symbol: "€", format: "%v %s" },
+const currenciesEnglish = {
+  EUR: { symbol: "€", format: "%s %v" },
   USD: { symbol: "$", format: "%s %v" },
   BRL: { symbol: "R$", format: "%s %v" },
   XOF: { symbol: "CFA", format: "%s %v" },
-  DKK: { symbol: "kr.", format: "%v %s" }
+  DKK: { symbol: "kr.", format: "%s %v" }
 };
 
 const getCurrencyFormat = currency => ({
-  ...numberFormat,
-  ...currencies[currency]
+  ...numberFormatEnglish,
+  ...currenciesEnglish[currency]
 });
 
 export function toAmountString(amount, currency) {
```

Comment:
what if i want to get the currency formated in fr? 
Suggestion: toAmountString(amount, currency=€, language=english)
the helper module could then be separated into a currency format or language module.